### ISSUE TITLE
normalize mishandles multi-word numbers

### ISSUE THREAD

**Issue by ChanceNCounter** [MycroftAI/lingua-franca#135]
```python3
>>> parse.normalize("four thirty seven")
'4 30 7'
```

**Comment by ChanceNCounter** [comment]
First thought: `extract_numbers(..., output_tokens=False)`

```python3
>>> parse.extract_numbers("four thirty seven", lang='en-us', output_tokens=True)
{'4': [0], '37': [1,2]}
```

`normalize()` could make use of this, which might, in turn, help the datetime extractors.

**Comment by ChanceNCounter** [comment]
I'm now realizing that the `extract_numbers()` *helper* already returns tokens. It's just that `extract_numbers()` only returns the tokens' values (the numbers.)

The question now is which way to call it in the normalizer, use the helper function or expose it via `extract_numbers()`?

**Comment by JarbasAl** [comment]
i had this fixed before, but change was rejected because core tests expect it.

i can try to find the PR with relevant discussion, i believe that was before LF.

But essentially some skills might expect it was the reasoning...

**Comment by JarbasAl** [comment]
here was my initial change, from when i added the extract_numbers method

https://github.com/MycroftAI/mycroft-core/pull/1867/files#diff-3dfdc2a85f591a9c2217a2e90730805bR1132

this was not final and not really the best implementation, but normalize was so basic in number handling that this covered a bunch of cases and still improved results

If i recall correctly it caused issues in an official skill and was later reverted (need to search PR)


It is my opinion that this is desired behavior and skill should be fixed instead of "maintaining the bug", but could be made configurable and that's where i was going with the Normalizer class  https://github.com/MycroftAI/lingua-franca/pull/55

**Comment by krisgesling** [comment]
This is a tough one, when I try it out loud, for me it all comes down to the length of silence between words.
- "four thirty seven" = 4 37
- "four, thirty, seven"  = 4 30 7
  (but STT isn't going to return punctuation like this)
- "nineteen twenty" = 19 20 or 1920
   but the difference is pretty trivial and agree it should remain separated then be handled by the higher level function eg `count()` vs `get_year()`

Reading a list of numbers seems to illustrate the problem best:
> Hey Mycroft multiply these numbers together, 4, 30, 7

But in practice I can't actually put a long enough gap in my speech to make it clear without the mic activation timing out. Also personally it seems like 4:37 is going to be a more common need.

@JarbasAI I take it you've hit this issue in developing Skills and needed to work around it?

**Comment by JarbasAl** [comment]
this made LF unusable in chatterbox (without wrappers that is)

here is an example (which involves other bugs fixed in #146 

- extract_datetime("twenty two minutes")  -> None
- extract_datetime("20 2 minutes") -> 2 minutes
